### PR TITLE
Implements "Log to the Unix syslog" example

### PR DESCRIPTION
**Pull request by budziq**
Please see https://github.com/brson/rust-cookbook/issues/61

### CODE REVIEW

File: src/app.md [hunk 1]
```diff
@@ -134,7 +134,31 @@ Your favorite number must be 256.
 
 [![log-badge]][log] [![syslog-badge]][syslog] [![cat-debugging-badge]][cat-debugging]
 
-[Write me!](https://github.com/brson/rust-cookbook/issues/61)
+Messages are logged to [UNIX syslog]. Logger backend is initialized
+with [`syslog::init`].
+[`syslog::Facility`] indicates type of program submitting log.
+[`log::LogLevelFilter`] denotes alowed log verbosity
```
Comment (dtolnay) [inline src/app.md]: "alowed" should be "allowed"

File: src/app.md [hunk 2]
```diff
@@ -134,7 +134,31 @@ Your favorite number must be 256.
...
+and `Option<&str>` holds optional aplication name.
```
Comment (dtolnay) [inline src/app.md]: "aplication" should be "application"

File: src/app.md [hunk 3]
```diff
@@ -134,7 +134,31 @@ Your favorite number must be 256.
...
+
+```rust,no_run
```
Comment (dtolnay) [inline src/app.md]: Why is this `no_run`? Is there some reason it would fail if run in CI?

Comment (budziq) [inline src/app.md]: Thanks for your time :) 
it is marked as `no_run` in order not to clobber the syslog. I guess that most people would prefer not to have noise in their syslog at every testrun.

**Comment by budziq** [comment]
@dtolnay typos fixed (sorry for that :disappointed:). The example itself is still `no_run` unless it is necessary ;)

**Pull request merged**